client-linuxapp/src/serviceinfo.rs

```diff
@@ -69,6 +74,54 @@ fn create_user(user: &str) -> Result<()> {
     Ok(())
 }
 
+fn create_user_with_password(user: &str, password: &str) -> Result<()> {
+    // Checks if user already present
+    let user_info = passwd::Passwd::from_name(user);
+    if user_info.is_some() {
+        log::info!("User {} is already present", user);
```

Comment:
This is just a nit: rust allows to do `log::info!("{user}");` (and with the `format!` macro, `println!` and some others) which in this case is equivalent to `log::info!("{}", user);`, but it is considered to be more clean, if you can use inline format parameters, you are encouraged to use them (see [this](https://doc.rust-lang.org/stable/std/fmt/)).

Note that it doesn't work if you need to do something complex with the variable before:
```
// this does not compile:
format!("Error creating new user with password: {sshkey_user.as_ref().unwrap()}");
```

Reply:
Thank you for sharing! I just updated my code to use the inline format where I could